sdk/storage/azure-storage-queue/tests/settings/service_versions.py

```diff
@@ -23,5 +23,5 @@ class ServiceVersion(str, Enum):
     "V2020_06_12": ServiceVersion.V2020_06_12,
     "V2020_08_04": ServiceVersion.V2020_08_04,
     "LATEST": ServiceVersion.V2020_08_04,
-    "LATEST_PLUS_1": ServiceVersion.V2020_06_12
+    "LATEST_PLUS_1": ServiceVersion.V2020_06_12,
```

Comment:
Sort of unrelated to this change but can you see if this file is being used anywhere? I don't think it is and if not, it should just be deleted.